pkg/asset/agent/manifests/releaseimagearch.go

```diff
@@ -0,0 +1,156 @@
+package manifests
```

Comment:
@andfasano I think this is what you were going for, can I get a sanity check before I continue?